examples/guides/se-restful-webservice/README.adoc

```diff
@@ -675,169 +580,125 @@ curl -X PUT http://localhost:8080/greet/greeting/%20
 Our code to update the greeting accepts this and saves it as the new greeting.
 --
 
-. Ping the health check endpoint again with the same command as before
+. Ping the health check endpoint again with the same command as before.
 +
 --
 [source,bash]
-curl -i -X GET http://localhost:8080/alive
+curl -i -X GET http://localhost:8080/health
 
-This time you should see
+This time you should see these two parts of the output indicating that something is
+wrong:
 [listing]
-{"error":"greeting is not set or is empty"}
-
-and with the `-i` added to the `curl` command you would see the 500 status returned.
---
-// end::tryLiveness[]
-
-== Add metrics support
-// tag::metricsIntro[]
-As a simple illustration of using metrics, we revise our greeting service to count how many times
-a client sends a request to the app.
-// end::metricsIntro[]
-
-. Add the metrics dependency to `pom.xml`
-+
---
-[source,xml]
-// _include::0-3:{pom}[tag=metricsDependency]
 ----
-        <dependency>
-            <groupId>io.helidon.metrics</groupId>
-            <artifactId>helidon-metrics</artifactId>
-        </dependency>
+{"outcome":"DOWN",...
+{"name":"greetingAlive","state":"DOWN","data":{"greeting":"not set or is empty"}}
 ----
+and with the `-i` added to the `curl` command you would see status 503 "Service
```

Comment:
If you add `-i` to the `curl` command, the status 503 "Service Unavailable" is returned.

Reply:
done